### ISSUE TITLE
get_residual_cnn not defined

### ISSUE THREAD

**Issue by MartinThoma** [cesarsalgado/tensorflowed#1]
In `models/residual_nets.py:135` you have the line `logits = get_residual_cnn(10, 128, 1, 28, 16, 3, 1)` but `get_residual_cnn` seems not to be defined.

**Comment by cesarsalgado** [comment]
My code is a mess right now. It is very coupled with some private code for my research. I will try to clean it as soon as possible.

Regarding get_residual_cnn, I forgot to change its name to get_cnn.

Thanks for reporting!

**Comment by cesarsalgado** [comment]
@MartinThoma I cleaned the repo. You can find an example on examples/mnist.py
